lib/bd_lint/validator.rb

```diff
@@ -0,0 +1,15 @@
+module BdLint
+  class Validator
+    GIT_HOOK_PATH = ".git/hooks/pre-commit".freeze
+
+    class GitHookError < StandardError; end
+
+    def self.check
+      raise GitHookError, "Please run `bundle exec rake bd_lint:generate:pre_commit`" if invalid?
```

Comment:
It looks like this task isn't defined for rails projects -- that logic might be backwards in the task definitions?

If that is how it is supposed to work, we should reflect that logic here.